gammapy/irf/tests/test_irf_write.py

```diff
@@ -63,11 +63,11 @@ def test_container_to_fits(self):
 
         assert self.aeff.to_fits().header['EXTNAME'] == 'EFFECTIVE AREA'
         assert self.edisp.to_fits().header['EXTNAME'] == 'ENERGY DISPERSION'
-        assert self.bkg.to_fits().header['EXTNAME'] == 'BACKGROUND'
+        assert self.bkg.to_hdulist()[1].header['EXTNAME'] == 'BACKGROUND'
```

Comment:
@cdeil 
This test file I am not sure to really understand the necessity. Me I am testing already the write and read method in my ```test_background.py```

Reply:
I don't think this is important. It's just trying to establish the bahaviour of to_hdulist, i.e. what it produces. Just keep as-is?